src/main/java/com/todolist/app/springmvcboottodolist/models/Item.java

```diff
@@ -0,0 +1,82 @@
+package com.todolist.app.springmvcboottodolist.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Item {
```

Comment:
I guess we can introduce lombok here to facilitate getters and setters work in pojos https://projectlombok.org/